optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfiguration.java

```diff
@@ -154,7 +153,8 @@ public SolverConfig solverConfig() {
                 // Return a mock ConstraintVerifier so not having ConstraintProvider doesn't crash tests
                 // (Cannot create custom condition that checks SolverConfig, since that
                 //  requires OptaPlannerAutoConfiguration to have a no-args constructor)
-                final String noConstraintProviderErrorMsg = "ConstraintVerifier is only supported for ConstraintProviders";
+                final String noConstraintProviderErrorMsg =
+                        "Cannot provision a ConstraintVerifier because there is ConstraintProvider class.";
```

Comment:
Should be "Cannot provision a ConstraintVerifier because there is no ConstraintProvider class."